app/components/DepositWithdraw/DepositWithdrawAssetSelector.js

```diff
@@ -15,6 +15,7 @@ class DepositWithdrawAssetSelector  extends React.Component {
 
     render(){
         const { props } = this;
+        const { include } = props;
```

Comment:
Change this to 
`const { include, openLedgerBackedCoins, rudexBackedCoins, blockTradesBackedCoins } = this.props`

Then remove the line before and update L53 correspondently.

Reply:
I've noticed you've worked a bit different than what I've seen on most other places of the app. First setting props and state. At least add the coins to the props line and update L53 for consistency.

Reply:
Good catch